platform/osx/os_osx.mm

```diff
@@ -1816,8 +1826,8 @@ virtual void log_error(const char *p_function, const char *p_file, int p_line, c
 			case CURSOR_FORBIDDEN: [[NSCursor arrowCursor] set]; break;
 			case CURSOR_VSIZE: [[NSCursor resizeUpDownCursor] set]; break;
 			case CURSOR_HSIZE: [[NSCursor resizeLeftRightCursor] set]; break;
```

Comment:
Might be worth changing V/H resize cursors to the one without middle line (with the fallback to the current), and add some missing:

```obj-c++
			case CURSOR_FORBIDDEN: [[NSCursor operationNotAllowedCursor] set]; break;
			case CURSOR_VSIZE: [cursorFromSelector(@selector(_windowResizeNorthSouthCursor)) set]; break; //+FALLBACK
			case CURSOR_HSIZE: [cursorFromSelector(@selector(_windowResizeEastWestCursor)) set]; break; //+FALLBACK

...

			case CURSOR_HELP: [cursorFromSelector(@selector(_helpCursor)) set]; break;
```

There's also `busyButClickableCursor` for WAIT/BUSY but IIRC Apple discourages direct use of spinning cursors.